MonopolyGame/src/cs414/a4/monopoly/backEnd/GameController.java

```diff
@@ -0,0 +1,54 @@
+package cs414.a4.monopoly.backEnd;
+
+import java.util.ArrayList;
+import java.util.List;
+import cs414.a4.monopoly.frontEnd.*;
+
+public class GameController {
+
+	private Game game;
+	private Board board;
+	private List<Player> players;
+	private Bank bank;
+	
+	public GameController(List<String> playerNames) {
+		game = new Game(playerNames);
+		board = game.setup();
+		players = game.getPlayers();
+		bank = Bank.getInstance();
+	}
+	
+	public void printPlayerInfo(){
+		for(Player player : players){
+			System.out.println("Player: " + player.getName() + "\nToken: " + 
+		player.getToken() + "\nPosition: " + 
+					board.getBoardSpaces()[player.getLocation()].getName() +
+					"\nProperties: " + player.propertiesOwned.size() + "\n");
+		}
+	}
+	
+	public void setPlayerTokens(List<String> tokens){
+		game.setPlayerTokensInOrder(tokens);
+	}
+	
+	public List<String> getPlayerNames(){
+		return game.getPlayerNamesInOrder();
+	}
+	
+	public static void main(String [] args){
+		//Create view and get names from view
+		ViewController view = new ViewController();
+		List<String> playerNames = view.getPlayerNames();
+		
+		GameController gameController = new GameController(playerNames);
+		playerNames = gameController.getPlayerNames();

```

Comment:
Not sure why you are resetting playerNames here. You create the gameController with playerNames, but then overwrite playerNames with what you just set controller to. Maybe you are trying to get them to point to the same place in memory? Just a comment that i'm not sure what's going on here.